tools/lib.sh

```diff
@@ -75,3 +75,7 @@ function contains_element() {
   return 1
 }
 
+function validate_config() {
+  log "Validating json config file $1"
+  python -m json.tool $1 1>/dev/null 
```

Comment:
I'm not sure if we want to do this. If we invoke `osqueryd`, `initDaemon` will try to create a pidfile, then will fail if the make package script was not run as root.

Meh, I could just remove this check if that is preferred.